package.json

```diff
@@ -2,13 +2,9 @@
   "name": "build-canada-bills",
   "version": "0.1.0",
   "private": true,
-  "engines": {
```

Comment:
@xrendan Why get rid of `engines`? We track this in `.node_version` and `.nvmrc` with it pinned. Would it be helpful to change it to `>=20.19.4`